homepage/templates/faq.html

```diff
@@ -0,0 +1,67 @@
+
+                    <h2 id="q-slug">I get an error when using Orange.</h2>
+                    <p class="indent">First, make sure you are using the latest version of Orange. If this doesn’t fix the issue, you should be able to see an error window pop up, prompting you to submit the issue to the developers. Please use this option as it allows us to recreate the issue and fix it. If this is not possible, please report the issue to our <a href="https://github.com/biolab/orange3/issues">issue tracker</a>.</p>
+
+                    <h2 id="q-slug">I believe there’s a bug in Orange.</h2>
+                    <p class="indent">First, make sure you are using the latest version of Orange. If the bug is still present, please submit an issue to our <a href="https://github.com/biolab/orange3/issues">issue tracker</a>. Follow the template and describe the issue as well as possible.</p>
+
+                    <p class="indent">Before submitting a new issue, check if someone has already reported the same problem. Upvote the issue with a +1 and/or add your comment. Only consider opening a new issue if you cannot find a corresponding existing open or closed issue.</p>
+
+                    <h2 id="q-slug">There’s a feature I wish to see in Orange.</h2>
+
+                    <p class="indent">Please submit a pull request with the feature. We welcome all contributions! Once you've made a PR, developer(s) will leave comments and when everything is fixed and tested, the pull request will be merged into Orange. See our <a href="https://orange.biolab.si/contributing/">Contribute guide</a> for more information.</p>
+
+                    <p class="indent">If you are not a developer, you can submit a feature request to our <a href="https://github.com/biolab/orange3/issues">issue tracker</a>. Describe what you wish to see in Orange and, if possible, provide a visual example.</p>
+
+                    <p class="indent">Before submitting a new feature request, check if someone has already requested the same feature. Upvote the request with a +1 and/or add your comment. Only consider opening a new request if you cannot find a corresponding existing request.</p>
+
+                    <h2 id="q-slug">I would like to learn how to use Orange. Where can I find the resources?</h2>
+                    <p class="indent">Widget documentation is available on our <a href="https://orange.biolab.si/toolbox/">website</a>. You can also access the same documentation directly from Orange by selecting the widget and pressing F1.</p>
+
+                    <p class="indent">We also provide an e-mail course (<a href="https://biolab.us15.list-manage.com/subscribe?u=2cad1f1cbf3a532a77efe4354&id=6f257f15ff">subscribe here</a>), beginner-level <a href="https://www.youtube.com/playlist?list=PLmNPvQr9Tf-ZSDLwOzxpvY-HrE0yv-8Fy">YouTube tutorials</a> and a regularly updated <a href="https://blog.biolab.si/">blog</a>.</p>
```

Comment:
You see the ampersand here is red because special characters like it in SGML attribute values should be escaped, i.e. `&` → `&amp;`.